### ISSUE TITLE
Each class is properly represented in both the training and test set?

### ISSUE THREAD

**Issue by izxi** [izxi/Learning#16]
When selecting data, so as to assure that each class is properly represented in both the training and test set, which of the following is used?

Select one:

- [x] - A. Cross validation
- [ ] - B. Stratification
- [ ] - C. Verification
- [ ] - D. Bootstrapping

**Comment by izxi** [comment]
**Cross-validation** is a technique for evaluating ML models by training several ML models on subsets of the available input data and evaluating them on the complementary subset of the data. Use cross-validation to detect overfitting, ie, failing to generalize a pattern.

**Comment by johnnieng** [comment]
B. Stratification *

**Comment by mebriki** [comment]
[Stratification](https://towardsdatascience.com/what-is-stratified-cross-validation-in-machine-learning-8844f3e7ae8e)
